static/app/components/events/eventEntries.tsx

```diff
@@ -340,12 +340,13 @@ const EventEntries = ({
         />
       )}
       {event.userReport && group && (
-        <StyledEventUserFeedback
-          report={event.userReport}
-          orgId={orgSlug}
-          issueId={group.id}
-          includeBorder={!hasErrors}
-        />
+        <EventDataSection title="User Feedback" type="user-feedback">
+          <StyledEventUserFeedback
```

Comment:
is this used in other places where it needs the padding?

Reply:
Actually turns out this didn't need a styled component after all, the other styles weren't doing anything. Just using EventUserFeedback now.